src/components/game.js

```diff
@@ -104,27 +129,45 @@ class Game extends React.Component {
       }
     }
 
+    const newMovedList = [];
+
     const mergeTiles = (tileMergeIntoPosX, tileMergedIntoPosY,
       tileMergedFromPosX, tileMergedFromPosY) => {
       temporaryBoard[tileMergeIntoPosX][tileMergedIntoPosY].value *= 2;
       temporaryBoard[tileMergeIntoPosX][tileMergedIntoPosY].isUsed = true;
       temporaryBoard[tileMergedFromPosX][tileMergedFromPosY].value = 0;
+      // eslint-disable-next-line max-len
+      newMovedList[newMovedList.length - 1].toCoords = new Coordinates(tileMergeIntoPosX, tileMergedIntoPosY);
```

Comment:
korzystasz dużo z newMovedList[newMovedList.length - 1], może wyciągnąć to do zmiennej? Pozwoli to też na skrócenie długości linii

Reply:
Korzystam tylko 3 razy. A niestety żeby to działało tak jak mówisz musiałbym mieć zmienną poza funkcją albo przekazywać ją jako argument do tych funkcji. Myślę, że niepotrzebne jest wprowadzanie takiej komplikacji 😄

Reply:
czy
```const aaa = newMovedList[newMovedList.length - 1];```
nie będzie działało?